indexer_app/handler.py

```diff
@@ -420,11 +420,42 @@ async def handle_streamer_message(streamer_message: near_primitives.StreamerMess
                             break
                         case "owner_add_admins":
                             logger.info(f"adding admins.. {args_dict}")
+                            if not match_nadabot_registry_pattern(receiver_id):
+                                break
                             await handle_add_nadabot_admin(args_dict, receiver_id)
                             break
-                        case "admin_add_whitelisted_deployers":
-                            logger.info(f"adding whitelisted deployers... {args_dict}")
-                            await handle_add_factory_deployers(args_dict, receiver_id)
+                        case (
+                            "admin_set_require_whitelist"
+                            | "admin_add_whitelisted_deployers"
+                            | "admin_set_protocol_config"
+                            | "admin_set_protocol_fee_recipient_account"
+                            | "admin_set_protocol_fee_basis_points"
+                        ):
+                            if not match_pot_factory_pattern(receiver_id):
+                                break
+                            logger.info(f"updating configs.. {args_dict}")
+                            await handle_set_factory_configs(args_dict, receiver_id)
+                            break
+                        case (
+                            "owner_set_admins"
+                            | "owner_clear_admins"
+                        ):
+                            if not match_pot_factory_pattern(receiver_id):
+                                break
+                            logger.info(f"updating admins.. {args_dict}")
+                            await handle_factory_admin_accounts(args_dict, receiver_id, True)
+                            break
+                        case "owner_add_admins":
+                            if not match_pot_factory_pattern(receiver_id):
+                                break
+                            logger.info(f"adding factory admins.. {args_dict}")
+                            await handle_factory_admin_accounts(args_dict, receiver_id, False)
+                            break
+                        case "owner_remove_admins":
+                            if not match_pot_factory_pattern(receiver_id):
+                                break
+                            logger.info(f"removing admins.. {args_dict}")
+                            await handle_factory_remove_admin(args_dict, receiver_id)
                             break
```

Comment:
@Prometheo do you want to just roll these up into the `handle_set_factory_configs` case? not strictly necessary but `admins` is a field on the config so these are kind of redundant now.